tests/Meilisearch.Tests/KeyTests.cs

```diff
@@ -0,0 +1,119 @@
+namespace Meilisearch.Tests
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using Xunit;
+
+    [Collection("Sequential")]
+    public class KeyTests : IAsyncLifetime
+    {
+        private IndexFixture fixture;
+        private MeilisearchClient client;
+
+        public KeyTests(IndexFixture fixture)
+        {
+            this.fixture = fixture;
+            this.client = fixture.DefaultClient;
+        }
+
+        public async Task InitializeAsync()
+        {
+            await this.fixture.DeleteAllIndexes(); // Test context cleaned for each [Fact]
```

Comment:
Shouldn’t we destroy all keys after that also?

Reply:
Hum, it would be better I think. An issue to think about for every SDK however. 
But this is not an emergency since the keys only impact the tests regarding the keys. But it can definitely bring some corner cases in the future, so I would say, yes, in the future, we should delete the keys we've created after the key tests.